LLMonFHIR/Settings/Prompt.swift

```diff
@@ -46,7 +44,9 @@ enum Prompt: String {
     
     var prompt: String {
         var prompt = UserDefaults.standard.string(forKey: rawValue) ?? defaultPrompt
-        prompt += Locale.preferredLanguages[0]
+        var promptTranslation = String(localized: "FHIR_PROMPT_TRANSLATION \("%@")")
+        promptTranslation = promptTranslation.replacingOccurrences(of: Prompt.promptPlaceholder, with: Locale.preferredLanguages[0])
+        prompt += promptTranslation
```

Comment:
You can probably compress the three lines down to the following statement:
```suggestion
        prompt += String(localized: "FHIR_PROMPT_TRANSLATION \(Locale.preferredLanguages[0])") 
```
Compared to the use case with the prompt where we allow the user to adjust the prompt in the settings, we don't offer that customization with the translation statement and therefore can just use normal string interpolation for localized strings to enable the same functionality. J

ust to be sure, please test this in the application so this works as expected.